executor/sort_test.go

```diff
@@ -44,3 +44,22 @@ func (s *testSuite4) TestSortRand(c *C) {
 		"      └─TableScan_6 10000.00 cop table:t, range:[-inf,+inf], keep order:false, stats:pseudo",
 	))
 }
+
+func (s *testSuite4) TestSortCorrelatedColumn(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("drop table if exists t1;")
+	tk.MustExec("create table t1(a int, b int);")
+	tk.MustExec("drop table if exists t2;")
+	tk.MustExec("create table t2(a int, b int);")
+
+	tk.MustQuery("explain select * from t1 where t1.a in (select t2.a as a from t2 where t2.b > t1.b order by t1.b);").Check(testkit.Rows(
```

Comment:
it's better to move this test to `explain-test`

Reply:
Would you mean all the tests in this file or just this test?